pkgs/racket-doc/scribblings/foreign/define.scrbl

```diff
@@ -97,3 +105,33 @@ with @racket[#:make-fail] or @racket[#:default-make-fail] in
 Equivalent to @racket[(provide (protect-out provide-spec ...))]. The
 @racket[provide-protected] identifier is useful with
 @racket[#:provide] in @racket[define-ffi-definer].}
+
+@section{FFI Symbol Conventions}
+
+@defmodule[ffi/unsafe/define/conventions]
+
+This module provides several
+@deftech{FFI identifier conventions} for use with
+@racket[#:make-c-id] in @racket[define-ffi-definer]. A
+@tech{FFI identifier convention} is any
+@tech[#:doc '(lib "scribblings/reference/reference.scrbl")]{syntax transformer}
+that converts one identifier to another.
+
+@history[#:added "6.9.0.5"]
+
+@defidform[convention:hyphen->underscore]{
+
+A convention that converts hyphens in an identifier to
+underscores.
+}
+
+@racketblock[
+  (define-ffi-definer define-gtk gtk-lib
+    #:make-c-id hyphen->underline)
```

Comment:
is `hyphen->underline` another convention?
EDIT: aha, it's a typo. This `racketblock` should be an `example`!

can you also give examples that just use the convention, e.g.

```
@examples[
  (hyphen->underline #'gtk-rc-parse)]
```